### ISSUE TITLE
Unable to find archetypes

### ISSUE THREAD

**Issue by mtn** [mtn/cocoa-eh-hugo-theme#21]
Hi!

Pretty new to Hugo, so bear with me if this is a stupid question. When I create new content, I encounter the warning:

`ERROR (date and time) Unable to find archetypes directory for theme "cocoa-eh"`

I understand that archetypes are somewhat like templates for metadata, but I don't think I have any need to use them. Is there any way I can suppress this error? Sorry about this not necessarily being theme-specific. :wolf:

**Comment by fuegoio** [comment]
Hi ! Thanks you for your issue !

I thought that archetypes were optionnals, but in fact, no. I will create a
post template too to have a useful archetypes.

**Comment by mtn** [comment]
Ah, I didn't notice "useful archetypes" so the pull request I made just adds a minimal empty one purely to suppress the error. With a blank default archetype, new content metadata includes a date and title. Is there anything you had in mind (for default or posts/blog content specifically)?

**Comment by fuegoio** [comment]
Great pull request ! Sorry to not have the time to do it right now, but a typical archetype is something like this : 

```
+++
date = "2016-12-18"
tags = ["blahblah"]
title = "Title"
description = "Desc"
meta_img = "/images/image.jpg"
+++
```

If you can't set it in your pull request I will merge it and see it later ! Thx :)

**Comment by mtn** [comment]
Addressed in #22 :hammer:

**Comment by fuegowolf** [comment]
Fix with PR #22 ! Thanks you again !
